tests/dummy/app/pods/docs/helpers/t/template.md

```diff
@@ -30,6 +30,25 @@ Compiles a [ICU message syntax](https://formatjs.io/docs/core-concepts/icu-synta
 }
 ```
 
+## Named VS Positional Arguments
```

Comment:
```suggestion
## Named vs. Positional Arguments
```

I'm not a native English speaker, but AFAIK the short form of `versus` would still be spelled as `vs.` in title case. What do you think?

Reply:
Good catch @buschtoens Buschtoens, actually I think neither is correct because this heading is title cased.

🤔 💭 Now that you bring it up, I'm considering this a bit more:
- using Latin terms isn't really a best practice in English technical writing, and
- also the word "versus" implies opposition, but named and positional arguments aren't opposed, or mutually exclusive in this case they can be used together

I will simplify this wording to avoid using "versus" entirely.

Reply:
😅 Writing docs is hard, thanks for the feedback @buschtoens !!!

1. I simplified the heading and final sentence of this section in https://github.com/ember-intl/ember-intl/pull/1633/commits/cd089913af14f51991e84d08313c269bad3c101e

2. I noticed that a couple of the other headings in the docs for the `t` helper were not title cased, which was inconsistent with the style of the rest of the docs, so I fixed that in https://github.com/ember-intl/ember-intl/pull/1633/commits/97ade66d33ad2fa12dc96e6b502d90e2a012c72b